elifetools/parseJATS.py

```diff
@@ -1,15 +1,14 @@
-from bs4 import BeautifulSoup
-from utils import *
-from utils_html import xml_to_html
-from json_rewrite import rewrite_json
-import rawJATS as raw_parser
-import re
 from collections import OrderedDict
-from six import iteritems
 
+from bs4 import BeautifulSoup
 from slugify import slugify
 
-from utils import unicode_value
+import elifetools.rawJATS as raw_parser
+import elifetools.json_rewrite
+from elifetools.utils import *
```

Comment:
`import *` is suspicious

Reply:
it was already there

Reply:
Yes, I am holding off on the refactor until the imports are sorted.